lmdk/cmake/build.cmake

```diff
@@ -68,10 +71,18 @@ foreach(MODULE ${MODULES_LIST})
 		#"-Wl,--no-undefined" "-Wl,--unresolved-symbols=report-all" "-Wl,--error-unresolved-symbols"
 		#"-Wl,--gc-sections"	# may remove .bss and that will result in rimage error, do not use for now
 		"-Wl,-Map,$<TARGET_FILE:${MODULE}>.map"	# optional: just for debug
-		"-T" "${MODULE}_ldscripts/elf32xtensa.x"
+		#"-T" "${MODULE}_ldscripts/elf32xtensa.x"
+		# FIXME: use a script to calculate values
+		"-Wl,-Ttext=0xa06ca000"
+		"-Wl,-Tdata=0xa06cb000"
+		"-Wl,--section-start=.rodata=0xa06cb100"
```

Comment:
magic - are we copied here ?

Reply:
there's a comment there - need a script to calculate these. The approach is rather simple: .text is fixed (per module? to make them unique?) the .data should then be page-aligned. Haven't been able to achieve that in a better way so far